app/views/shared/_consultees_table.html.erb

```diff
@@ -9,7 +9,7 @@
   </thead>
   <tbody class="govuk-table__body">
   <% @planning_application.planning_application_constraints.each do |constraint| %>
-    <tr class="govuk-table__row">
+    <tr class="govuk-table__row" id="constraint-<%= constraint.id %>">
```

Comment:
We should try to avoid injecting ERB in attributes because escaping inside HTML attributes is different from escaping HTML elements - always use a tag helper like this:

``` ruby
<%= tag.tr(id: "constraint-#{constraint.id}", class: "govuk-table__row") do %>
...
<% end %>
```

This ensures attributes are escaped properly and we don't inadvertently open an XSS hole.

Reply:
I actually added this in order to make the tests more specific (to pick the right row) and then ended up not using it, so I might as well just remove it.